main/src/ca/uwaterloo/flix/api/lsp/provider/completion/Completion.scala

```diff
@@ -203,13 +203,14 @@ sealed trait Completion {
         textEdit = TextEdit(context.range, name + " "),
         detail = None,
         kind = CompletionItemKind.Variable)
-    case Completion.EnumTagCompletion(enumSym, caseSym) =>
+    case Completion.EnumTagCompletion(enumSym, caseSym, snippet) =>
```

Comment:
Why do you do it like this? If you look at PredicateCompletion you can see that I simply store the arity in the completion.

The formatting should be done elsewhere, not in the Completer itself.